### ISSUE TITLE
[Bug]: Borders shown when they should not in 2022.5

### ISSUE THREAD

**Issue by TheFes** [piitaya/lovelace-mushroom#415]
### Current Behavior

In 2022.5 it seems that borders are shown in cards like `custom:vertical-stack-in-card` for mushroom cards:

Example code:
```yaml
square: true
columns: 2
type: grid
cards:
  - type: custom:vertical-stack-in-card
    cards:
      - type: custom:mushroom-entity-card
        entity: light.groep_woonkamer
      - type: custom:mushroom-entity-card
        entity: light.groep_woonkamer
  - type: custom:vertical-stack-in-card
    cards:
      - type: entity
        entity: light.groep_woonkamer
      - type: entity
        entity: light.groep_woonkamer
```

Result
![image](https://user-images.githubusercontent.com/28803438/165911221-1221d408-829a-4335-8564-909ea939a069.png)

The card using normal `entity` cards is rendered correct without borders, while the one with the mushroom cards still shows borders between the individual cards.

### Expected Behavior

No borders :)

### Steps To Reproduce

See code above

### Environment

```markdown
- OS: HA OS 8.0 rc3
- Broswer: Chrome
- Hassio: 2022.5.0b2
```

**Comment by acesyde** [comment]
Hi, 

vertical-stack-card search a `ha-card` in the shadow root node but with mushroom we have a `mushroom-card` before so it's doesn't work

**Comment by piitaya** [comment]
Fixed by #417
